gnowsys-ndf/gnowsys_ndf/es_initialize.py

```diff
@@ -0,0 +1,751 @@
+								# 	"analyzer": "reverse"
+								# }
+							}
+						}
+					}
+				},
+				"Group": {
+					"properties": {
+						"name": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"altnames":{
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"content": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"tags": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						}
+					}
+				},
+				"AttributeType": {
+					"properties": {
+						"name": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"altnames":{
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"content": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"tags": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						}
+					}
+				},
+				"GAttribute": {
+					"properties": {
+						"name": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						},
+						"tags": {
+							"type": "text",
+							"fields": {
+								"trigram": {
+									"type": "text",
+									"analyzer": "trigram"
+								},
+								# "reverse": {
+								# 	"type": "text",
+								# 	"analyzer": "reverse"
+								# }
+							}
+						}
+					}
+				}
+			}
+	}
+
```

Comment:
`request_body`:
- It's static JSON configuration.
- Can we have it as a separate file kept out of code and import in required file(`es_initialize.py`).